tutorials/clustering.md

```diff
@@ -64,14 +146,38 @@ using our informational views:
 SELECT * FROM timescaledb_information.data_node;
 ```
 
-Now that we've created a database and added a couple data nodes,
-let's go ahead and create a distributed hypertable. With the following
-commands, we'll create a distributed hypertable for collecting
-temperature readings from sensors:
+### Create a new role
 
+Now that you have created a database and added a couple data nodes, let's go
+ahead and create a new user role we can use for our distributed database.
+We'll use Timescale's `distributed_exec` function to perform this action on
+the data nodes:
```

Comment:
Entire tutorial uses `you`, but here it switches to `we`. May be to use `you` here too?

Reply:
Good catch, I've been trying to fix up these pronouns from the original tutorial.